mssql-ballerina/Client.bal

```diff
@@ -175,12 +175,21 @@ public type Options record {|
 
 # SSL configuration to be used when connecting to the MsSQL server
 #
-# + encrypt - encryption for all the data sent between the client and the server if the server has a certificate installed
-# + trustServerCertificate - If "true", the SQL Server SSL certificate is automatically trusted when the communication layer is encrypted using TLS
-
+# + integratedSecurity - Set to "true" to indicate that Windows credentials are used by SQL Server on Windows operating
+#                        systems. If "true", the local computer credential cache is searched for credentials that were
+#                        provided when a user signed in to the computer or network.
+# + encrypt - Encryption for all the data sent between the client and the server if the server has a certificate
+#             installed
+# + trustServerCertificate - If "true", the SQL Server SSL certificate is automatically trusted when the communication
+#                            layer is encrypted using TLS
+# + cert - Keystore configuration of the trust certificates
+# + hostNameInCertificate - The host name to be used in validating the SQL Server TLS/SSL certificate.
 public type SecureSocket record {|
+    boolean integratedSecurity?;
     boolean encrypt?;
     boolean trustServerCertificate?;
```

Comment:
what is the usage of the `trustServerCertificate` config? is this for oneway TLS.

Reply:
Yes it is

> Set to "true" to specify that the driver doesn't validate the server TLS/SSL certificate.

> If "true", the server TLS/SSL certificate is automatically trusted when the communication layer is encrypted using TLS.

> If "false", the driver validates the server TLS/SSL certificate. If the server certificate validation fails, the driver raises an error and closes the connection. The default value is "false". Make sure the value passed to serverName exactly matches the Common Name (CN) or DNS name in the Subject Alternate Name in the server certificate for a TLS/SSL connection to succeed. For more information about encryption support, see Understanding encryption support.

Reply:
seems like `encrpt` and `trustServerCertificate` going together. similar to `SSL_MODE` in other DB connectors